contractcourt/channel_arbitrator.go

```diff
@@ -2223,6 +2224,7 @@ func (c *ChannelArbitrator) channelAttendant(bestHeight int32) {
 			// close status of the channel.
 			err = c.cfg.MarkChannelClosed(
 				closeInfo.ChannelCloseSummary,
+				channeldb.ChanStatusLocalCloseInitiator,
```

Comment:
Any way to test this previous omission or is it too involved for the size of the PR?

Reply:
it's best we could use `channeldb.OpenChannel.HasChanStatus` to check the intended status was used, yet tracing down where to access a `OpenChannel`, `ChannelArbitrator -> ChannelArbitratorConfig -> Channel`, ends with an `ArbChannel` interface.

One way is to expand this interface so that we could add test in `channel_arbitrator_test.go`, but that's too heavy if it's only for testing.

Or we could test it in `chain_arbitrator_test.go`, by constructing local/remote/coop close event?